locations/spiders/hobby_lobby.py

```diff
@@ -0,0 +1,44 @@
+from urllib.parse import urlparse
+
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class HobbyLobbySpider(SitemapSpider, StructuredDataSpider):
+    name = "hobby_lobby"
+    allowed_domains = ["hobbylobby.com"]
+    item_attributes = {
+        "brand": "Hobby Lobby",
+        "brand_wikidata": "Q5874938",
+    }
+    sitemap_urls = ["https://www.hobbylobby.com/sitemap-Store.xml"]
+    sitemap_rules = [
+        (r"/stores/search/(\d+)$", "parse_sd"),
+    ]
+    time_format = "%H:%M %p"
```

Comment:
`%H` is for 24 hour time, `%I` is for 12 hour time.

You want this line to be:

`time_format = "%I:%M %p"`